packages/web/src/hooks/common/use-loading.tsx

```diff
@@ -1,41 +1,29 @@
-import { isEmptyObject } from "@utils/validation-utils";
 import { useEffect, useState } from "react";
 
-interface Props {
-  isLoading: boolean;
-  isFetching: boolean;
-  isBack: boolean;
-  status: boolean;
-  connected: boolean;
+interface UseLoadingProps {
+  shouldTrigger: boolean;
```

Comment:
`shouldTrigger` feels like the name of a function.
Is there a better name ?

Reply:
i will fix this

Reply:
@jinoosss i do some research and i think that this name is not bad at all, can i use it

Reply:
I think i will change it to `shouldTriggerLoading` instead, is it good ?

Reply:
In my opinion, `trigger` is sufficient. 
Because `trigger` is a boolean type.
If `trigger` is true, we think the `trigger` has been fired.

In that case, please change the function of `triggerLoading` to `startLoading`.